NEWS.md

```diff
@@ -31,6 +31,7 @@ OpenSSL 3.0
   * enable-crypto-mdebug and enable-crypto-mdebug-backtrace were mostly
     disabled; the project uses address sanitize/leak-detect instead.
   * Added OSSL_SERIALIZER, a generic serializer API.
+  * Added OSSL_PARAM_BLD, an easier to use API to OSSL_PARAM.
```

Comment:
Maybe a CHANGES.md entry as well, with a bit more fleshing out?

Reply:
Apart from RTFM, what else needs to be said?

Reply:
A short summary of the FM is often seen.  You might want to look at some entries in said file for inspiration.

Reply:
Added a note the CHANGES.md